customers/services/profile.py

```diff
@@ -0,0 +1,133 @@
+import json
+from dataclasses import dataclass
+from typing import Dict, Optional
+from uuid import UUID
+
+import requests
+from django.http import HttpRequest
+
+from utils.relay import from_global_id, to_global_id
+
+from ..models import CustomerProfile
+
+PROFILE_API_URL = "PROFILE_API_URL"
+PROFILE_TOKEN_SERVICE = "PROFILE_TOKEN_SERVICE"
+
+
+@dataclass
+class HelsinkiProfileUser:
+    id: UUID
+    first_name: str = ""
+    last_name: str = ""
+    email: str = ""
+    customer: Optional[CustomerProfile] = None
```

Comment:
Looks now great with `dataclass` 👍  One nit: having `customer` as field is not needed. By dropping it we could avoid the heavy customer table scan/iteration on line 89. what do you think?

Reply:
I agree, it's not necessary but I was thinking on a generic use case where you would need access to the customer's leases, invoices, or something else customer-related. As it's not required for now, we can drop it and add it back if it's needed at some point. 👍